app/models/katello/content_view.rb

```diff
@@ -650,6 +651,17 @@ def check_composite_action_allowed!(env)
       true
     end
 
+    def check_orphaned_content_facets(env)
```

Comment:
nit pick

```suggestion
    def check_orphaned_content_facets(envs)
```

or


```suggestion
    def check_orphaned_content_facets(environments: [])
```
to make it amply clear

Reply:
Updated with 2nd suggestion.